Sources/PointFree/Environment.swift

```diff
@@ -49,18 +49,22 @@ public struct AppEnvironment {
 
   public static func with(
     airtableStuff: @escaping AirtableCreateRow = AppEnvironment.current.airtableStuff,
+    createUser: @escaping CreateUser = AppEnvironment.current.createUser,
```

Comment:
do we wanna add `appEnv` to this? 

since it's so easy to lens into stuff I wonder if we stop maintaining arguments for every field in `with` and just require a whole new environment:

```swift
AppEnvironment.with(AppEnvironment.current |> \.appEnv .~ .development) {
  // stuff
}
```

Reply:
oh ha, `appEnv` is in `envVars` not the environment 👀 

so never mind the first part, but the second part stands...

Reply:
`appEnv` is on `envVars`, so it's more like:

``` swift
AppEnvironment.with(AppEnvironment.current |> \.envVars.appEnv .~ development) {
  // stuff
}
```

Reply:
Yup. Will remove!